client/data/sites/use-site-excerpts-query.ts

```diff
@@ -29,6 +30,15 @@ const fetchSites = (
 	} );
 };
 
+export const useSiteExcerptsQueryInvalidator = () => {
+	const queryClient = useQueryClient();
+	const invalidate = useCallback(
+		() => queryClient.invalidateQueries( { queryKey: [ USE_SITE_EXCERPTS_QUERY_KEY ] } ),
+		[ queryClient ]
+	);
+	return invalidate;
+};
+
 export const useSiteExcerptsQuery = (
```

Comment:
This hook is used [here](https://github.com/Automattic/wp-calypso/blob/fb3d890129047e0e9edd9e7a50d5b388ef7496af/client/sites/components/sites-dashboard.tsx#L125) to load the site list on the /sites page